core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -237,8 +233,6 @@ private void saveFirebaseInstanceIdIfUpdated(DeviceAnonymized deviceAnonymized,
 		}
 		deviceAnonymized.setFirebaseInstanceId(newFirebaseInstanceId);
 		deviceAnonymizedRepository.save(deviceAnonymized);
-		logger.info("Changed Firebase instance ID from {} to {}", oldFirebaseInstanceId.orElse("<not set>"),
```

Comment:
Why was this log message removed?

Reply:
It was added for debugging an has served its purpose. It doesn't add value anymore.